Don't retry in case of error

Nothing Drake loads should have nasty side-effects, so there should
never be any reason to crash and resume. Retrying might slow things
down, which we definitely do NOT want.

Reasoning for not upstreaming this patch: Drake-specific option value.

--- mypy/moduleinspect.py
+++ mypy/moduleinspect.py
@@ -143,5 +143,5 @@
         if isinstance(res, str):
             # Error importing module
-            if self.counter > 0:
+            if False:
                 # Also try with a fresh process. Maybe one of the previous imports has
                 # corrupted some global state.
